### PR TITLE
Support working with multiple Elasticsearch nodes

### PR DESCRIPTION
**Pull request by khaledk2**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -145,7 +145,12 @@ def get_index_data_from_database(resource="all"):
             resource_table_=None, re_create_index=True, only_values=False
         )
         # validat ethe indexing
-        test_indexing_search_query(deep_check=True, check_studies=True)
+        try:
+            test_indexing_search_query(deep_check=False, check_studies=True)
+        except Exception:
+            pass
```
Comment (joshmoore) [inline manage.py:151]: What does this mean? e.g. a node is not reachable?

Comment (khaledk2) [reply manage.py:151]: Sorry, I used this `try except` block for testing on my local machine. I will remove it.

**Comment by jburel** [comment]
the first 2 commits are in #59 I don't think they are needed for that work

**Review by jburel — CHANGES_REQUESTED** [review]
Could you adjust so it only contains the changes required for multi-nodes? 
This is what we need to be able to test and deploy this set of improvement.

**Comment by khaledk2** [comment]
Now, this PR contains only the required changes for deploying multiple elasticsearch nodes.

**Comment by jburel** [comment]
testing on 30.11 the docker image containing this collection of proposed changes

**Review by jburel — APPROVED** [review]
Proposed changes tested via https://github.com/IDR/deployment/pull/387

**Pull request merged**